examples/jsm/controls/ArcballControls.js

```diff
@@ -0,0 +1,2495 @@
+		this._manager.get( 'singlepan' ).recognizeWith( 'triplepan' );
+
+		this._manager.get( 'doublepan' ).recognizeWith( 'pinch' );
+		this._manager.get( 'doublepan' ).recognizeWith( 'rotate' );
+		this._manager.get( 'doublepan' ).recognizeWith( 'triplepan' );
+
+		this._manager.get( 'pinch' ).recognizeWith( 'rotate' );
+
+		this._manager.get( 'triplepan' ).recognizeWith( 'rotate' );
+		this._manager.get( 'triplepan' ).recognizeWith( 'pinch' );
+
+		this._manager.get( 'doubletap' ).recognizeWith( 'press' );
+
+
+		this._manager.get( 'triplepan' ).requireFailure( 'singlepan' );
+		this._manager.get( 'triplepan' ).requireFailure( 'pinch' );
+		this._manager.get( 'triplepan' ).requireFailure( 'rotate' );
+		this._manager.get( 'triplepan' ).requireFailure( 'doublepan' );
+
+		this._manager.get( 'rotate' ).recognizeWith( 'pinch' );
+
+		this.setCamera(camera);
+
+		if ( this.scene != null ) {
+
+			this.scene.add( this._gizmos );
+		
+		}
+
+		this._manager.on( 'press', this.onPress );
+		this._manager.on( 'doubletap', this.onDoubleTap );
+
+		this._manager.on( 'singlepanstart', this.onSinglePanStart );
+		this._manager.on( 'singlepanmove', this.onSinglePanMove );
+		this._manager.on( 'singlepanend', this.onSinglePanEnd );
+
+		this._manager.on( 'doublepanstart', this.onDoublePanStart );
+		this._manager.on( 'doublepanmove', this.onDoublePanMove );
+		this._manager.on( 'doublepanend', this.onDoublePanEnd );
+
+		this._manager.on( 'pinchstart', this.onPinchStart );
+		this._manager.on( 'pinchmove', this.onPinchMove );
+		this._manager.on( 'pinchend', this.onPinchEnd );
+
+		this._manager.on( 'rotatestart', this.onRotateStart );
+		this._manager.on( 'rotatemove', this.onRotateMove );
+		this._manager.on( 'rotateend', this.onRotateEnd );
+
+		this._manager.on( 'triplepanstart', this.onTriplePanStart );
+		this._manager.on( 'triplepanmove', this.onTriplePanMove );
+		this._manager.on( 'triplepanend', this.onTriplePanEnd );
+
+		this.domElement.addEventListener( 'mousedown', this.onMouseDown );
+		this.domElement.addEventListener( 'wheel', this.onWheel );
+
+		document.addEventListener( 'keydown', this.onKeyDown );
+		window.addEventListener( 'resize', this.onWindowResize );
+
+	};
+
+	//listeners
+
+	onWindowResize = () => {
+
+		const scale = ( this._gizmos.scale.x + this._gizmos.scale.y + this._gizmos.scale.z ) / 3; 
+		this._tbRadius = this.calculateTbRadius( this.camera );
+
+		const newRadius = this._tbRadius / scale;
+		const curve = new EllipseCurve( 0, 0, newRadius, newRadius );
+		const points = curve.getPoints( this._curvePts );
+		const curveGeometry = new BufferGeometry().setFromPoints( points );
+
+
+		for( let gizmo in this._gizmos.children ) {
+
+			this._gizmos.children[ gizmo ].geometry = curveGeometry;
+
+		}
+
+		this.dispatchEvent( _changeEvent );
+
+	};
+
+	onMouseDown = ( event ) => {
+
+		if ( event.button == 1 && this.enabled && this.enablePan ) {
+	
+			event.preventDefault();
```

Comment:
Please avoid the usage of `preventDefault()`. We also want to avoid these calls since they produce unexpected side effect for app level code.

Reply:
Do you have any alternatives to suggest? I would like to prevent window scrolling for operations which involves mouse wheel.